transmart-core-db/src/main/groovy/org/transmartproject/db/multidimquery/query/Constraint.groovy

```diff
@@ -281,6 +283,12 @@ class FieldConstraint extends Constraint {
     Object value
 
     static constraints = {
+        field validator: { val, obj, Errors errors ->
```

Comment:
I'm not sure what happens if the closure returns null, but at least if the relevant parameter ('field') is not present at all in the query, the @BindUsing won't be called and there won't be an error.